nautobot/core/templates/components/panel/stats_panel_body.html

```diff
@@ -0,0 +1,8 @@
+<div class="panel-body">
+    {% for key, value in stats.items %}
```

Comment:
is `key` unused here? Could maybe do `for url, value, title in stats.values` instead?